test/utils/utils.go

```diff
@@ -247,3 +244,51 @@ func UncommentCode(filename, target, prefix string) error {
 	// nolint:gosec
 	return os.WriteFile(filename, out.Bytes(), 0644)
 }
+
+// LoadImage does not work when using podman, so LoadImageArchive is used instead
+// refer to https://github.com/kubernetes-sigs/kind/issues/2038
+func LoadContainerImageToKindCluster(ctx context.Context,
+	kindCluster *kind.Cluster,
+	dockerImage string,
+	containerTool string) (context.Context, error) {
```

Comment:
ditto https://github.com/etcd-io/etcd-operator/pull/123#discussion_r2036963870
```suggestion
	containerTool string) error {
```